json/src/test/resources/com/esri/json/hadoop/unenclosed-json-esc-points.json

```diff
@@ -0,0 +1,15 @@
+{"attributes":{"rowid": 1505, "text": "\""},"geometry":{"x":15.0,"y":5.0}}
+{"attributes":{"rowid":  535, "text": "\'"},"geometry":{"x":5,"y":35}}
+{"attributes":{"rowid": 2323, "text": "\\"},"geometry":{"x":23,"y":23}}
+{"attributes":{"rowid": 3222, "text": "\}"},"geometry":{"x":32,"y":22}}
+{"attributes":{"rowid": 3728, "text": "\{"},"geometry":{"x":37,"y":28}}
+{"attributes":{"rowid": 2233, "text": "\,"},"geometry":{"x":22,"y":33}}
+{"attributes":{"rowid": 2838, "text": "\:"},"geometry":{"x":28,"y":38}}
+{"attributes":{"rowid": 3434, "text": "\1"},"geometry":{"x":34,"y":34}}
+{"attributes":{"rowid": 6219, "text": "\2"},"geometry":{"x":62,"y":19}}
+{"attributes":{"rowid": 7114, "text": "\3"},"geometry":{"x":71,"y":14}}
+{"attributes":{"rowid": 7525, "text": "\a"},"geometry":{"x":75,"y":25}}
+{"attributes":{"rowid": 6535, "text": "\b"},"geometry":{"x":65,"y":35}}
+{"attributes":{"rowid": 5549, "text": "\C"},"geometry":{"x":55,"y":49}}
+{"attributes":{"rowid": 6545, "text": "\D"},"geometry":{"x":65,"y":45}}
+{"attributes":{"rowid": 4566, "text": "x\"y\"z"},"geometry":{"x":45,"y":66}}
```

Comment:
The fact that these tests may contain invalid JSON, does not imply that we need support invalid JSON.